2.1/alpine/Dockerfile

```diff
@@ -78,6 +78,7 @@ RUN set -ex \
 		scanelf --needed --nobanner --recursive /usr/local \
 			| awk '{ gsub(/,/, "\nso:", $2); print "so:" $2 }' \
```

Comment:
Ah, it's this bit that causes the trouble -- we only know that we're linked against `libruby.so.2.3`, not the full path to the proper `.so` file. :disappointed:

```console
$ docker run -it --rm ruby:2.3-alpine sh
/ # scanelf --needed --nobanner --recursive /usr/local | awk '{ gsub(/,/, "\nso:", $2); print "so:" $2 }' | sort -u
so:libc.musl-x86_64.so.1
so:libcrypto.so.1.0.0
so:libffi.so.6
so:libgdbm.so.4
so:libgdbm_compat.so.4
so:libncursesw.so.6
so:libreadline.so.6
so:libruby.so.2.3
so:libssl.so.1.0.0
so:libyaml-0.so.2
so:libz.so.1
/ # 
```